src/services/kratos/private.ts

```diff
@@ -0,0 +1,29 @@
+import { getKratosConfig } from "@config"
+import { Configuration, V0alpha2Api, V0alpha2ApiInterface } from "@ory/client"
+
+import { MissingExpiredAtKratosError } from "./errors"
+
+const { publicApi, adminApi } = getKratosConfig()
+
+const KratosSdk: (kratosEndpoint: string) => V0alpha2ApiInterface = (kratosEndpoint) =>
+  new V0alpha2Api(new Configuration({ basePath: kratosEndpoint }))
+
+export const kratosPublic = KratosSdk(publicApi)
+export const kratosAdmin = KratosSdk(adminApi)
+
+export const toDomainSession = (session: KratosSession) => {
```

Comment:
Definitely need to declare the return type here IMO - inference could cause issues.